webpack/dev.config.js

```diff
@@ -30,7 +30,16 @@ module.exports = {
         contentBase: path.resolve(__dirname, '../.tmp'),
         watchContentBase: true,
         port: 9000,
-        historyApiFallback: true
+        historyApiFallback: true,
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+        },
+        proxy: {
+            '/api': {
```

Comment:
seems demo should work without proxy, need to have option on set domain in all form requests

Reply:
lets separate from select fixes